netbout-client/src/main/java/com/netbout/client/RtAttachment.java

```diff
@@ -119,7 +119,17 @@ public boolean unseen() throws IOException {
 
     @Override
     public Date date() throws IOException {
-        return new Date();
+        return new Date(
```

Comment:
@CarlosGines I agree that his is not specific to the task, I even tried to point this out :)

You know I just think it's more in line with the spirit of how we work to put a puzzle here because:
1. We have some problem with duplication (not a big one, but still)
2. This PR makes it slightly worse ( again not your fault, fix is fine here ), so this PR certainly would need to have that duplication cleaned up to be "perfectly fine".
3. Hence the puzzle would make this PR "perfect" if solved.

... believe me I would create the bug and rake in the 15 min if I thought it was appropriate ;)